docs/checkers/index.md

```diff
@@ -9,6 +9,31 @@ email to \[david at cs dot toronto dot edu\].
 These errors generally indicate a misuse of variables, control flow, or other Python features in our
 code.
 
+(W0134)=
+
+### Return in finally
+
+This error occurs when a `return` statement is used inside a `finally` block.
+
+```{literalinclude} /../examples/pylint/w0134_return_in_finally.py
+
+```
+
+Corrected Version:
+
+```python
+def error_code():
+
+    error_codes = [0, 1, 2]
+
+    try:
+        return error_codes[2]
+    except IndexError:
+        ...
```

Comment:
I would modify this example to capture a more typical "intended" behaviour as follows:

- Make `error_codes` a function parameter (in both the correct version and original example)
- Keep the return statement in the `except`
- Remove the return statement in the `try` (perhaps replace it with a `print` or something), and instead make the bottom return statement `return error_codes[2]`, which is what the "typical" behaviour would be.

Reply:
I just made the modifications and pushed the changes.